languages/cpp/maintainers.md

```diff
@@ -10,3 +10,4 @@ TODO: add senior maintainers
 
 - Sebastian Borza (Slack: `@sebito91`, Github: [sebito91](https://github.com/sebito91))
 - Rohit Goswami (Slack: `@HaoZeke`, Github: [HaoZeke](https://github.com/HaoZeke))
+- AeroStun (Slack: `@Aerostun`, Github: [AeroStun](https://github.com/AeroStun))
```

Comment:
Full name?

Reply:
I don't want this information publicly disclosed

Reply:
@iHiD any thoughts on the above? ^^

Reply:
(Note that I'm completely fine with sharing that information behind closed doors)

Reply:
I'll defer to @iHiD, but I'd imagine that it's fine to omit someone's full name. This seems more like a shout-out/thank you to contributors which need not be public.
Thus far Rust has done this in their [`maintainers.md`](https://github.com/exercism/v3/blob/master/languages/rust/maintainers.md), and one maintaier did this on the [C track](https://github.com/exercism/v3/blob/master/languages/c/maintainers.md).

Reply:
> This seems more like a shout-out/thank you to contributors which need not be public.

Yep. This.

Thanks for pinging me to check though :)